### ISSUE TITLE
Allow lock tabs

### ISSUE THREAD

**Issue by disgustingdev** [marta-file-manager/marta-issues#511]
Allow lock tabs like in TotalCommander:
1. Lock without dir change (on opening dir opens new tab)
2. Lock with dir change (on opening dir opens in same tab, but after clicking on tab or reopening app tab will be on dir, with that it was locked)

**Comment by Saratxe** [comment]
Yes, I also miss this feature very much and would be happy to see it implemented.
